modules/web-console/web-console-server/src/main/resources/i18/errors.properties

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2019 GridGain Systems, Inc. and Contributors.
+#
+# Licensed under the GridGain Community Edition License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+err.sign-up-not-allowed=Sign-up is not allowed. Ask your administrator to create account for you.
+err.confirm-email=Confirm your email
+err.activation-not-enabled=Activation was not enabled!
+err.too-many-activation-attempts=Too many activation attempts
+
+err.db-lost-connection-during-tx=Database connection was lost during transaction
+err.active-tx-not-found=No active transaction was found
+err.data-access-violation=Data access violation
+err.db-not-available=Database not available
+
+err.prohibited-revoke-admin-rights=Self revoke of administrator rights is prohibited
+err.parse-signin-req-failed=Failed to parse signin request
+err.could-not-create-hash=Could not create hash
+err.tokens-no-specified-in-agent-handshake-req=Tokens not specified in agent handshake request
+err.failed-auth-with-tokens=Failed to authenticate with token(s): {0}
```

Comment:
Do we need indexed placeholders?

Reply:
Spring MessageSource using https://docs.oracle.com/javase/7/docs/api/java/text/MessageFormat.html under the hood.